snowplow_tracker/emitters.py

```diff
@@ -68,6 +69,8 @@ def __init__(
         on_failure: Optional[FailureCallback] = None,
         byte_limit: Optional[int] = None,
         request_timeout: Optional[Union[float, Tuple[float, float]]] = None,
+        retry_codes={},
+        max_retry=60,
```

Comment:
I am also thinking that we should probably introduce a `buffer_capacity` configuration option. The problem is that currently if the collector will not respond for a longer time, the buffer will be forever increasing without any limit. This can be quite dangerous and cause the app to crash. We should put a limit on when to step adding events to the buffer. See how this is implemented in the Java tracker.

While doing that, we should probably also rename `buffer_size` to `batch_size` (similar to Java) because having two configurations `buffer_size` and `buffer_capacity` would be too confusing.